crates/sparrow-batch/src/batch.rs

```diff
@@ -94,6 +94,52 @@ impl Batch {
         })
     }
 
+    #[cfg(any(test, feature = "testing"))]
+    pub fn record_batch(self) -> Option<RecordBatch> {
```

Comment:
How does this differ from `into_record_batch`? Can it use that?

Reply:
`into_record_batch` requires you to get/know the schema. This is just a helper that looks at the `data` data type and creates the schema for you inline. 

Though, did add comment that this is a "hack" and we want to standardize on Batch.